src/layer/types.js

```diff
@@ -15,7 +15,7 @@ export type LayersManagerPropsT = {
 };
 
 export type LayersContextT = {
-  host: typeof undefined | ?HTMLElement,
```

Comment:
We need to keep the `typeof undefined | ?HTMLElement` type here, since in the case when `LayersManager` is not added in the app the host value will be `undefined` and we'll render the content in the `document.body` instead.

Reply:
Isn’t `typeof undefined` a subset of `?HTMLElement` though?
https://flow.org/en/docs/types/maybe/

`?HTMLElement` is equivalent to `HTMLElement & typeof undefined & null` already.